test/prow/fluid/fluidapi.py

```diff
@@ -214,7 +214,7 @@ def assemble_dataset(testcase):
     elif testcase == "juicefs-minio":
         mount = Mount()
         mount.set_mount_info("juicefs-community", "juicefs:///")
-        mount.add_options("bucket", "https://%s:9000/minio/test" % minio_svc)
+        mount.add_options("bucket", "http://%s:9000/minio/test" % minio_svc)
```

Comment:
The reason of changing https to http? Using http protocol is insecure.

Reply:
The default minio yaml in prow-test cluster is using http not https. If minio is newly created for the first time, the e2e test-case juicefs_minio.py test will fail. I have already told this problem to Zhu Weiwei and Xu Zhihao.

Should I change back http to https?